app/config.py

```
@@ -22,6 +22,8 @@ class Settings(BaseSettings):
     days_in_advance: int = 7
     max_tee_times_per_day: int = 2
 
+    scheduler_api_key: str = ""
```

Empty string default means the server starts "successfully" but with broken auth. The verify_scheduler_api_key function does check for this and returns 500, but should the application fail fast at startup if this critical security config is missing in production?